Makefile

```diff
@@ -23,7 +23,11 @@
 generate-bindings:
 	cargo run -p generate-bindings -- --target ../../packages/_generated/commontypes/src
 
+generate-change-json:
+	cargo run -p crate-extractor -- create -m crates/contract/Cargo.toml -o packages/_generated/change/change.json
```

Comment:
For config files like this its sometiemes a bit clearer to use the longhand commandline options. Although we don't do this in other places. Longer commands in Make can also be multiline:

something like this should work:

```
generate-change-json:
	cargo run --package crate-extractor -- create \
	  --manifest crates/contract/Cargo.toml \
	  --output packages/_generated/change/change.json
```

Reply:
We could also add commit the `_generated/change` folder and add a minimal package.json.
So we have keep overview of what we generate. Maybe we can generate to src and gitignore this. (Like in bindings)